### PR TITLE
Inclusion of diffusers/dreambooth example with IPEX

### PR DESCRIPTION
**Pull request by nazneenn**
## Type of Change

Example code 

## Description

Dreambooth is a method customize text-to-image models like Stable Diffusion with a few (3-5) images of a subject.
This example illustrates how to finetune DreamBooth with the CompVis/stable-diffusion-v1-4 model, utilizing IPEX.

## Expected Behavior & Potential Risk

NA

## How has this PR been tested?

This example has been tested on an SPR machine

## Dependency Change?

torch and ipex version - http://mlpc.intel.com/downloads/cpu/2023/2023_ww44_Internal/

### CODE REVIEW

File: examples/pytorch/diffusion_model/diffusers/dreambooth/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+accelerate>=0.16.0
+transformers>=4.25.1
+ftfy
+tensorboard
+Jinja2
+http://mlpc.intel.com/downloads/cpu/2023/2023_ww44_Internal/torch-2.2.0a0+git9955283-cp38-cp38-linux_x86_64.whl
```
Comment (yiliu30) [inline examples/pytorch/diffusion_model/diffusers/dreambooth/requirements.txt:6]: Are these available to the public now? If so, suggest utilizing the release version.

Comment (nazneenn) [reply examples/pytorch/diffusion_model/diffusers/dreambooth/requirements.txt]: Hi @yiliu30 
updated the requirements.txt file with the public version

File: examples/pytorch/diffusion_model/diffusers/dreambooth/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+accelerate
+transformers
+ftfy
+tensorboard
+Jinja2
+torch --index-url https://download.pytorch.org/whl/cpu
```
Comment (yiliu30) [inline examples/pytorch/diffusion_model/diffusers/dreambooth/requirements.txt:6]: @chensuyue , do we need to add them to BoM?

**Review by yiliu30 — APPROVED** [review]
LGTM

**Pull request merged**